client/my-sites/themes/current-theme/button.jsx

```diff
@@ -16,15 +16,23 @@ export default class extends React.Component {
 		icon: PropTypes.string.isRequired,
 		href: PropTypes.string,
 		onClick: PropTypes.func,
+		isPrimary: PropTypes.bool,
 	};
 
 	render() {
 		return (
 			<a
 				role="button"
-				className={ classNames( 'current-theme__button', 'current-theme__' + this.props.name, {
-					disabled: ! this.props.href,
-				} ) }
+				className={ classNames(
```

Comment:
Is there a reason this uses custom markup rather than the existing `Button` component? 🤔 That could handle a lot of this logic, I think: https://github.com/Automattic/wp-calypso/tree/99a3eb38b0cd27ad35391c02e5729c0aa28dfcaa/packages/components/src/button

Reply:
Good call; migrating to use that is definitely a lot cleaner. I'll push up an update.

Reply:
I'll hold off on a final review until that's done. Thanks Roz!